Classes/Tools.php

```diff
@@ -33,7 +33,7 @@
  *
  * @license http://www.gnu.org/licenses/gpl.html GNU General Public License, version 3 or later
  */
-abstract class Tools
+class Tools
```

Comment:
Ahh sorry I see what you mean... yes. lets do that in an Update class. much cleaner. I will make changes.